src/bot.ts

```diff
@@ -1,11 +1,10 @@
 import { strict as assert } from 'assert'
 import TelegramBot, { Message } from 'node-telegram-bot-api'
-
-const debug = require('debug')('DEBUG-bot')
-const info = require('debug')('INFO-bot')
+import Logger from './util/Logger'
 
 require('dotenv').config()
 
+const log = new Logger('bot')
```

Comment:
do you need to instantiate Logger every time, potentially in every file if there will be more. Wouldn't a singleton or static methods be better